zha/zigbee/cluster_handlers/smartenergy.py

```diff
@@ -320,14 +320,7 @@ async def async_update(self) -> None:
             for a in self.REPORT_CONFIG
             if a["attr"] not in self.cluster.unsupported_attributes
         ]
-        result = await self.get_attributes(attrs, from_cache=False, only_cache=False)
-        if result:
-            for attr, value in result.items():
-                self.attribute_updated(
-                    self.cluster.find_attribute(attr).id,
-                    attr,
-                    value,
-                )
+        await self.get_attributes(attrs, from_cache=False, only_cache=False)
```

Comment:
Coverage for this (SmartEnergy Metering sensor polling) added here:
- https://github.com/zigpy/zha/pull/359